src/coreComponents/managers/Functions/TableFunction.hpp

```diff
@@ -21,10 +21,15 @@
 
 #include "common/EnumStrings.hpp"
 #include "managers/Functions/FunctionBase.hpp"
+#include "LvArray/src/tensorOps.hpp"
 
 namespace geosx
 {
 
+static constexpr localIndex maxDimensions = 4;
```

Comment:
Could you put this back into `TableFunction` scope (as a public static member)? `TableFunctionKernelWrapper` can still reference it then, but it won't just sit in global `geosx` namespace.

Reply:
I put the `maxDimensions` parameter back to the `TableFunction` class so it is not in the global `geosx` namespace anymore.